### ISSUE TITLE
数据集

### ISSUE THREAD

**Issue by Zhaihanlei** [lalithjets/Surgical_VQA#1]
Hi, where can I download the images in the dataset?The link you provided is only Q&A pairs。Thanks for your answer

**Comment by lalithjets** [comment]
Hi, 

Sorry for the late reply, I lost track of the message. I will keep a constant watch moving forward.

**(1) EndoVis18:** The official challenge website seems to have some errors. The following links were used to download the dataset. Do remember to cite the challenge paper (https://arxiv.org/pdf/2001.11190.pdf) if you use the images:
  - Train set 1, 1st half: https://mega.nz/#!tiY2xSZI!Tc6yqUI50Xh9A0LiSdFdNm0guu5rFCDtqKT05firvyU
  - Train set 1, 7 corrected frames: https://mega.nz/#!46YwBCZS!EolKDSipkmGOskDrSk51N8a7yeaGdAJF3zdY05nUbDs
  - Train set 1, 2nd half: https://mega.nz/#!Q2IACYKY!LUGQHxLGKUZuXlc6nnwOCD8eBxoapyINkaBhVy1Rb80
  - Train set 2, 1st half: https://mega.nz/#!s6ZwiAgS!bbrehM3IwTHrFnrkMI2MxW5blrpzspfbj6GGwBETe8k
  - Train set 2, 2nd half: https://mega.nz/#!5uRy1AZA!E6IdQQrfnp5LfjQB6nTD1mq1T2u5myk2StUEZem1M90
  - Test set: https://drive.google.com/drive/folders/1dySa4sIVUysAApKHqDzcyGxin8tAhfUi?usp=sharing

**(2) Cholec80:** You can request and download from: http://camma.u-strasbg.fr/datasets

Thank you.

**Comment by ThompsonISAT** [comment]
> Hi,
> 
> Sorry for the late reply, I lost track of the message. I will keep a constant watch moving forward.
> 
> **(1) EndoVis18:** The official challenge website seems to have some errors. The following links were used to download the dataset. Do remember to cite the challenge paper (https://arxiv.org/pdf/2001.11190.pdf) if you use the images:
> 
> * Train set 1, 1st half: https://mega.nz/#!tiY2xSZI!Tc6yqUI50Xh9A0LiSdFdNm0guu5rFCDtqKT05firvyU
> * Train set 1, 7 corrected frames: https://mega.nz/#!46YwBCZS!EolKDSipkmGOskDrSk51N8a7yeaGdAJF3zdY05nUbDs
> * Train set 1, 2nd half: https://mega.nz/#!Q2IACYKY!LUGQHxLGKUZuXlc6nnwOCD8eBxoapyINkaBhVy1Rb80
> * Train set 2, 1st half: https://mega.nz/#!s6ZwiAgS!bbrehM3IwTHrFnrkMI2MxW5blrpzspfbj6GGwBETe8k
> * Train set 2, 2nd half: https://mega.nz/#!5uRy1AZA!E6IdQQrfnp5LfjQB6nTD1mq1T2u5myk2StUEZem1M90
> * Test set: https://drive.google.com/drive/folders/1dySa4sIVUysAApKHqDzcyGxin8tAhfUi?usp=sharing
> 
> **(2) Cholec80:** You can request and download from: http://camma.u-strasbg.fr/datasets
> 
> Thank you.

Hi, I have the same question! Could you also provide the Med_vqa dataset? I have to download the images from https://github.com/abachaa/VQA-Med-2019, but there are no /img_features/5x5/synpic39156.hdf5 files.
Thank you so much for this!

**Comment by Parlin-Majere** [comment]
> > Hi,
> > Sorry for the late reply, I lost track of the message. I will keep a constant watch moving forward.
> > **(1) EndoVis18:** The official challenge website seems to have some errors. The following links were used to download the dataset. Do remember to cite the challenge paper (https://arxiv.org/pdf/2001.11190.pdf) if you use the images:
> > 
> > * Train set 1, 1st half: https://mega.nz/#!tiY2xSZI!Tc6yqUI50Xh9A0LiSdFdNm0guu5rFCDtqKT05firvyU
> > * Train set 1, 7 corrected frames: https://mega.nz/#!46YwBCZS!EolKDSipkmGOskDrSk51N8a7yeaGdAJF3zdY05nUbDs
> > * Train set 1, 2nd half: https://mega.nz/#!Q2IACYKY!LUGQHxLGKUZuXlc6nnwOCD8eBxoapyINkaBhVy1Rb80
> > * Train set 2, 1st half: https://mega.nz/#!s6ZwiAgS!bbrehM3IwTHrFnrkMI2MxW5blrpzspfbj6GGwBETe8k
> > * Train set 2, 2nd half: https://mega.nz/#!5uRy1AZA!E6IdQQrfnp5LfjQB6nTD1mq1T2u5myk2StUEZem1M90
> > * Test set: https://drive.google.com/drive/folders/1dySa4sIVUysAApKHqDzcyGxin8tAhfUi?usp=sharing
> > 
> > **(2) Cholec80:** You can request and download from: http://camma.u-strasbg.fr/datasets
> > Thank you.
> 
> Hi, I have the same question! Could you also provide the Med_vqa dataset? I have to download the images from https://github.com/abachaa/VQA-Med-2019, but there are no /img_features/5x5/synpic39156.hdf5 files. Thank you so much for this!

I think the problem is that the dataloader still defaults to the ones for processing videos instead of images, which they used feature_extraction_(corresponding dataset).py to extract hdf5 files for training. As for VQA-Med-2019, a dataset full of images I don't think there is a way to do so. You have to do some change to the current method to get it running which is my guess.

**Comment by lalithjets** [comment]
Hi Parlin,

yes, you are right.

Hi ThompsonISAT,
For the Med-VQA, we used the raw dataset (image, question, and answer) from https://github.com/abachaa/VQA-Med-2019 and MedFuseNet from https://github.com/dhruvsharma15/MEDVQA. However, we pre-processed the images for feature extraction. The feature extraction code sample is available inside the dataset folder. 

Thank you

**Comment by Parlin-Majere** [comment]
> Hi Parlin,
> 
> yes, you are right.
> 
> Hi ThompsonISAT, For the Med-VQA, we used the raw dataset (image, question, and answer) from https://github.com/abachaa/VQA-Med-2019 and MedFuseNet from https://github.com/dhruvsharma15/MEDVQA. However, we pre-processed the images for feature extraction. The feature extraction code sample is available inside the dataset folder.
> 
> Thank you

But regarding Med-VQA, I modified a parsing script so to parse the images of Med-VQA all into one size and converted into hdf5 format for training, yet there are still some errors regarding the masks I think. How did you guys do it back then?

Edited: So do you mean that for VQA section, you are using MedFuseNet from said repo?
